api/organisations/tasks.py

```diff
@@ -73,3 +86,85 @@ def finish_subscription_cancellation():
     ):
         subscription.organisation.cancel_users()
         subscription.save_as_free_subscription()
+
+
+def send_admin_api_usage_notification(
+    organisation: Organisation, matched_threshold: int
+) -> None:
+    """
+    Send notification to admins that the API has breached a threshold.
+    """
+    recipient_list = list(
+        FFAdminUser.objects.filter(
+            userorganisation__organisation=organisation,
+            userorganisation__role=OrganisationRole.ADMIN,
+        ).values_list("email", flat=True)
+    )
+
+    context = {
+        "organisation": organisation,
+        "matched_threshold": matched_threshold,
+    }
+    send_mail(
+        subject=f"Flagsmith API use has reached {matched_threshold}%",
+        message=render_to_string("organisations/api_usage_notification.txt", context),
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        recipient_list=recipient_list,
+        fail_silently=True,
+    )
```

Comment:
We may need to discuss this, but we should use an HTML template here.

Reply:
Ok this is implemented now, though as we discussed more follow on work will come with better visuals for our emails.